### PR TITLE
Update rl_cartpole_ray to use toolkit and framework argument and remo…

### PR DESCRIPTION
**Pull request by goelakash**
…ve docker image URI

*Issue #, if available:*

*Description of changes:*
Updates the code in the Jupyter notebook for rl_cartpole_ray to replace the `image_uri` param with RL toolkit and framework params instead (Ray 0.8.5 with Tensorflow)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by yijiezh — APPROVED** [review]
LGTM

File: reinforcement_learning/rl_roboschool_stable_baselines/rl_roboschool_stable_baselines.ipynb [hunk 1]
```diff
@@ -424,6 +424,13 @@
     "                <source src=\"data:video/mp4;base64,{0}\" type=\"video/mp4\" />\n",
     "             </video>'''.format(encoded.decode('ascii')))"
    ]
+  },
+  {
+   "cell_type": "code",
```
Comment (aaronmarkham) [inline reinforcement_learning/rl_roboschool_stable_baselines/rl_roboschool_stable_baselines.ipynb:429]: Could remove this file from the PR.

File: reinforcement_learning/rl_tic_tac_toe_coach_customEnv/rl_tic_tac_toe_coach_customEnv.ipynb [hunk 2]
```diff
@@ -314,7 +308,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.2"
+   "version": "3.6.10"
```
Comment (aaronmarkham) [inline reinforcement_learning/rl_tic_tac_toe_coach_customEnv/rl_tic_tac_toe_coach_customEnv.ipynb:311]: Could remove this file from the PR.

**Review by aaronmarkham — DISMISSED** [review]
Looks good. Could remove a couple of files from the PR that were just run, but no real changes made.

**Comment by aaronmarkham** [comment]
CI failed due to hitting a capacity limit... being looked into now.

File: reinforcement_learning/rl_roboschool_stable_baselines/rl_roboschool_stable_baselines.ipynb (hunk 1 above)
Comment (goelakash) [reply reinforcement_learning/rl_roboschool_stable_baselines/rl_roboschool_stable_baselines.ipynb:429]: Thanks, removed.

File: reinforcement_learning/rl_tic_tac_toe_coach_customEnv/rl_tic_tac_toe_coach_customEnv.ipynb (hunk 2 above)
Comment (goelakash) [reply reinforcement_learning/rl_tic_tac_toe_coach_customEnv/rl_tic_tac_toe_coach_customEnv.ipynb]: Removed.

**Comment by goelakash** [comment]
Closing this PR in favour of https://github.com/aws/amazon-sagemaker-examples/pull/1877 
There are some minor differences and the SDK v2 rollback in the `common` folder code has been avoided.

**Pull request closed without merging**